### PR TITLE
Ulimit

### PR DESCRIPTION
**Pull request by deimosfr**
Hi,

It can be usefull to update the ulimit option. I've added a template for it to be both compatible Debian and RedHat.

Thanks

### CODE REVIEW

File: tasks/configuration.yml [hunk 1]
```diff
@@ -45,3 +46,7 @@
     backup=yes
   notify: restart rabbitmq-server
   when: rabbitmq_conf_env is defined
+
+- name: set ulimit
+  template: src=rabbitmq_default.j2 dest=/etc/default/rabbitmq-server owner=rabbitmq group=rabbitmq mode=0755
```
Comment (Mayeu) [inline tasks/configuration.yml]: Can you make this multi-line instead of everything on only one line? It really improve readability.

**Comment by Mayeu** [comment]
Hello,

Thank you for this pull request, please see my comment inline, also, please separate this commit from the one adding the rabbitmq tags in the `install.yml` file.

I'll merge this when those change are made :)

**Comment by deimosfr** [comment]
I've made changes, please tell me if it's ok now

**Comment by Mayeu** [comment]
Thank you for your contribution.

Sadly, and has you may have guessed, I don't use this project anymore, nor do I have the time to continue to maintain it. I may at some point start to use it again, but that is not on my priority right now.

Feel free to fork and improve, I can even link potential fork in the README.

Wishing you the best, and a gajillion more contribution in your lifetime 🙏✨

**Pull request closed without merging**